docs/website/docs/v1.x/dsl/socket/websocketframe.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "WebSocketFrame"
+sidebar_label: "WebSocketFrame"
+---
+In the [WebSocket](https://datatracker.ietf.org/doc/html/rfc6455) protocol, communication happens using frames. ZIO
+HTTP's [WebSocketFrame](https://github.com/dream11/zio-http/blob/main/zio-http/src/main/scala/zhttp/socket/WebSocketFrame.scala)
+is its representation of those frames. The protocol defines the following types of frames:
```

Comment:
check grammar 
```suggestion
is representation of those frames. The Domain defines the following types of frames:
```

Reply:
@amitksingh1490 The grammar for the sentence seems correct. However, using `domain` instead of `protocol` makes more sense.